content/css/jenkins.css

```diff
@@ -1268,27 +1268,41 @@ a {
   color:#069;
 }
 
-#toc-blog{margin-left: -1rem;}
+#toc-blog {
+  margin-left: -1rem;
+}
+
 .section.blogs h4.blog-title {
-    font-size: 1.25rem;
-    padding-bottom: 0.25rem;
+  font-size: 1.25rem;
+  padding-bottom: 0.25rem;
 }
+
 .section.blogs .content {
-  padding-top:.5rem
+  padding-top: .5rem
 }
+
 .container.blog-post {
-  min-height:65rem;
+  min-height: 65rem;
 }
+
 .container.blog-post h1 {
-  font-size:2.25rem;
-  margin-bottom:1.rem;
+  font-size: 2.25rem;
+  margin-bottom: 1.rem;
 }
-.container.blog-post .submitted {display:block; margin:1rem 0}
+
+.container.blog-post .submitted {
+  display: block;
+  margin: 1rem 0;
+}
+
 .section.blogs h4.blog-title {
-    font-size: 1.25rem;
-    padding-bottom: 0.25rem;
+  font-size: 1.25rem;
+  padding-bottom: 0.25rem;
 }
 
+.container.blog-post .main-content {
+  margin-bottom: 60px
```

Comment:
Agreed that it looks better, good idea. 
Though why not just put "some" space, like a line or two of "blank text" equivalent, using e.g. `2em` instead of fixed size?